### ISSUE TITLE
--full-paths error: require(…) is not a function

### ISSUE THREAD

**Issue by nvartolomei** [milankinen/livereactload#112]
I'm getting this error when using https://github.com/agraboso/redux-api-middleware as dependency of my project.

Any ideas how to fix this?

Cross-issue: https://github.com/agraboso/redux-api-middleware/issues/76

**Comment by milankinen** [comment]
Hi @nvartolomei!

Is this error happening during the initial page render or after a reload? Is it possible to see the source file using that `redux-api-middleware`? Does the problem disappear if you remove that middleware from your codebase? 

Is your project OS so that I could take a closer look at this one?

**Comment by nvartolomei** [comment]
@milankinen will try to move `examples/redux` to a point where it will fail, hope can do it :)

**Comment by milankinen** [comment]
Cool! Thanks

**Comment by nvartolomei** [comment]
Finally, after 2 hours of messing up with this I found that adding `--full-paths` option to watchify makes it fall apart.

```patch
--- a/examples/02-redux/package.json
+++ b/examples/02-redux/package.json
@@ -8,28 +8,31 @@
     "bundle:prod": "browserify site.js -t babelify -g envify -o bundle.js",
     "watch": "npm run watch:server & npm run watch:bundle & wait",
     "watch:server": "nodemon --exec babel-node --ignore bundle.js -- server.js",
-    "watch:bundle": "watchify site.js -v -t babelify -g envify -p livereactload -o bundle.js"
+    "watch:bundle": "watchify site.js -v -t babelify -g envify -p livereactload -o bundle.js --full-paths"
```

**Comment by milankinen** [comment]
Nice, thanks! I fixed that already once but perhaps I've broken `fullPaths` again somehow... 😬 

If not using `fullPaths`, then everything works fine right? I'll take a look at this.

**Comment by nvartolomei** [comment]
Yes, it works if fullPaths option is disabled.

**Comment by milankinen** [comment]
Fixed in `3.0.0`.
